src/components/Navbar/Navbar.js

```diff
@@ -82,6 +105,47 @@ export const Navbar = () => {
           )}
         </div>
       </nav>
+      <div
+        className={showSearchModal ? "search-modal-bg" : "search-modal-bg hide"}
+      >
+        <div
+          className={
+            showSearchModal
+              ? "slide-modal search-modal slide-modal-show"
+              : "search-modal slide-modal"
+          }
+        >
+          <div className="search-modal-top">
+            <p className="search-modal-title">
+              Search videos related to coding.
+            </p>
+            <IoClose onClick={toggleSearchModal} />
+          </div>
+          <div className="">
+            <form
+              className="search-mobile nav-form"
+              onSubmit={e => e.preventDefault()}
+            >
+              <input
+                type="search"
+                placeholder="search..."
+                className="search-input"
+                value={searchQuery || ""}
```

Comment:
The initial value of search query is `" "` so it won't be undefind I guess.

Reply:
i did this to avoid warning related to uncontrolled input